relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -128,6 +132,7 @@ pub(crate) fn extract_span_tags(event: &mut Event, config: &Config) {
     // TODO: To prevent differences between metrics and payloads, we should not extract tags here
     // when they have already been extracted by a downstream relay.
     let shared_tags = extract_shared_tags(event);
+    let sdk_name = event.sdk_name().to_owned();
```

Comment:
nit: It would be slightly cheaper to set `let is_mobile = MOBILE_SDKS.contains(event.sdk_name())` here, and plug that into `extract_tags`.

Or even reuse the value that `extract_shared_tags` wrote to `shared_tags.get(SpanTagKey::Mobile)`.

Reply:
good idea! just passing an is_mobile boolean now https://github.com/getsentry/relay/pull/2761/files#diff-313f39c52849630c10c4f6da270c2ed0f99d7deac8894a838bd166dddc94b19dR135